specification/automation/resource-manager/Microsoft.Automation/stable/2015-10-31/dscConfiguration.json

```diff
@@ -366,6 +372,34 @@
           },
           {
             "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "$filter",
```

Comment:
>$filter [](start = 21, length = 7)

What about adding the `x-ms-odata` extension?

Reply:
@RayStone1998 For your reference - https://github.com/Azure/azure-rest-api-specs/blob/master/specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNodeConfiguration.json#L254